etest/docker/container.py

```diff
@@ -16,7 +16,11 @@
 
 def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     """Commit a Docker container."""
-    repo = repository.replace("=", "").replace("[", "").replace("]", "")
+    print(repository)
+    repo = repository.replace("=", "")\
+                     .replace("[", "")\
+                     .replace("]", "")\
+                     .replace(",", "")
```

Comment:
This would probably be more clear as a filter now rather than chained replaces.  Especially with the strange newline escaping required.

Reply:
The newline thing isnt required, I added it since it was a massive line before.